generators/server/templates/src/main/resources/config/application-dev.yml.ejs

```diff
@@ -255,6 +255,11 @@ spring:
     cache: false
 <%_ if (serviceDiscoveryType || applicationTypeGateway || applicationTypeMicroservice) { _%>
   sleuth:
+    # https://github.com/spring-projects/spring-framework/issues/27844#issuescomment-1002866885
+    # https://github.com/spring-cloud/spring-cloud-sleuth/issues/2094
+    async:
+      enabled: false
+    enabled: true
```

Comment:
Not sure it should be enabled by default

Reply:
I agree we should disable it by default